classes/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] In the `book` class you should create a `rental` method and remember the `book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(person, date)
    Rental.new(date, person, self)
  end
```